### ISSUE TITLE
Github rate limit exceeded leading to failing e2e test suite

### ISSUE THREAD

**Issue by bkimminich** [juice-shop/juice-shop#498]
Protractor tests run into the following `Error: Failed to make Github request, rate limit reached.` sometimes. This could be a problem from Pull Requests not inheriting the GitHub token as was reported in other cases, e.g. https://github.com/minishift/minishift/issues/502 or https://github.com/jspm/github/issues/67.

```
> juice-shop@7.0.0 update-webdriver /home/travis/build/bkimminich/juice-shop
> webdriver-manager update
[15:44:12] I/file_manager - creating folder /home/travis/build/bkimminich/juice-shop/node_modules/protractor/node_modules/webdriver-manager/selenium
(node:6645) UnhandledPromiseRejectionWarning: Error: Failed to make Github request, rate limit reached.
    at Request.req.on.response (/home/travis/build/bkimminich/juice-shop/node_modules/protractor/node_modules/webdriver-manager/built/lib/binaries/config_source.js:155:28)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at Request.onRequestResponse (/home/travis/build/bkimminich/juice-shop/node_modules/request/request.js:1068:10)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at HTTPParser.parserOnIncomingClient (_http_client.js:551:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:115:23)
    at TLSSocket.socketOnData (_http_client.js:440:20)
    at emitOne (events.js:116:13)
(node:6645) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:6645) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[15:44:12] I/update - chromedriver: unzipping chromedriver_2.36.zip
[15:44:13] I/update - chromedriver: setting permissions to 0755 for /home/travis/build/bkimminich/juice-shop/node_modules/protractor/node_modules/webdriver-manager/selenium/chromedriver_2.36
> juice-shop@7.0.0 protractor /home/travis/build/bkimminich/juice-shop
> node test/e2eTests.js
Server listening on port 3000
(node:6667) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
[15:44:16] I/launcher - Running 1 instances of WebDriver
[15:44:16] I/direct - Using ChromeDriver directly...
[15:44:16] E/direct - Error code: 135
[15:44:16] E/direct - Error message: Could not find update-config.json. Run 'webdriver-manager update' to download binaries.
[15:44:16] E/direct - Error: Could not find update-config.json. Run 'webdriver-manager update' to download binaries.
  at Direct.getNewDriver (/home/travis/build/bkimminich/juice-shop/node_modules/protractor/built/driverProviders/direct.js:63:31)
  at Runner.createBrowser (/home/travis/build/bkimminich/juice-shop/node_modules/protractor/built/runner.js:195:43)
  at q.then.then (/home/travis/build/bkimminich/juice-shop/node_modules/protractor/built/runner.js:339:29)
  at _fulfilled (/home/travis/build/bkimminich/juice-shop/node_modules/q/q.js:834:54)
  at self.promiseDispatch.done (/home/travis/build/bkimminich/juice-shop/node_modules/q/q.js:863:30)
  at Promise.promise.promiseDispatch (/home/travis/build/bkimminich/juice-shop/node_modules/q/q.js:796:13)
  at /home/travis/build/bkimminich/juice-shop/node_modules/q/q.js:556:49
  at runSingle (/home/travis/build/bkimminich/juice-shop/node_modules/q/q.js:137:13)
  at flush (/home/travis/build/bkimminich/juice-shop/node_modules/q/q.js:125:13)
  at _combinedTickCallback (internal/process/next_tick.js:131:7)
  at process._tickCallback (internal/process/next_tick.js:180:9)
  at Function.Module.runMain (module.js:695:11)
  at startup (bootstrap_node.js:188:16)
  at bootstrap_node.js:609:3
[15:44:16] E/launcher - Process exited with error code 135
Protractor exited with code 135 (FAILED)
```

**Comment by bkimminich** [comment]
Should be resolved by e76838a8374443e9bae53466e990cfeea7de721d. Closing for now.

**Comment by bkimminich** [comment]
https://github.com/angular/webdriver-manager/issues/216
